_data/projects/internet-speed-testing.yml

```
@@ -0,0 +1,20 @@
+---
+name: Internet speed testing
+desc: >-
+  Android library for test internet speed implement by kotlin language!
+site: https://github.com/oatrice/internet-speed-testing
+tags:
+- android
+- library
```

This is the reason why CI is currently failing:

```
/home/runner/work/up-for-grabs.net/up-for-grabs.net/_data/projects/internet-speed-testing.yml - 
Tag verification failed!
Use 'libraries' instead of library
```

```suggestion
- libraries
```

---

Thank you so much.:+1::+1::+1: